src/test/java/com/trello/boards/NewBoardTest.java

```diff
@@ -0,0 +1,124 @@
+        ChromeDriver driver = new ChromeDriver();
+
+        List<WebElement> openMemberMenu = driver.findElementsByXPath(".//button[contains(@aria-label, 'Open Member Menu')]");
+
+        if (openMemberMenu.size() > 0) {
+            openMemberMenu.get(0).click();
+            driver.findElementByLinkText("Sign Off").click();
+        }
+    }
+
+    @Test
+    public void searchBoardFieldCheck() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
+        ChromeDriver driver = new ChromeDriver();
+        WebElement searchField = driver.findElementByXPath(".//input[@name= 'search-boards']");
+
+        driver.get("https://trello.com/");
+        driver.findElementByLinkText("Log In").click();
+        driver.findElementById("user").sendKeys("fesenko.b@icloud.com");
+        driver.findElementById("password").sendKeys("B0r1sTr3ll0");
+        driver.findElementById("login").click();
+        Waiters.waitSeconds(4);
+        driver.findElementByXPath(".//button[@aria-label= 'Open Boards Menu']//span").click();
+
+        Assert.assertEquals(searchField.getAttribute("placeholder"), "Find boards by name…", "Placeholder is wrong");
+    }
+
+    @Test
+    public void userExistedBoardCheck() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
+        ChromeDriver driver = new ChromeDriver();
+        Actions action = new Actions(driver);
+
+        WebElement userBoard = driver.findElementByXPath(".//a[@title= 'Borys_Tech-Stack']");
+        String boardTitle = driver.findElementByXPath(".//input[@class= 'board-name-input js-board-name-input']").getAttribute("value");
+
+        driver.get("https://trello.com/");
+        driver.findElementByLinkText("Log In").click();
+        driver.findElementById("user").sendKeys("fesenko.b@icloud.com");
+        driver.findElementById("password").sendKeys("B0r1sTr3ll0");
+        driver.findElementById("login").click();
+        Waiters.waitSeconds(4);
+        driver.findElementByXPath(".//button[@aria-label= 'Open Boards Menu']//span").click();
+
+        Assert.assertTrue(userBoard.isEnabled(), "The board isn't found");
+
+        action.click(userBoard).build().perform();
+
+        Assert.assertEquals(boardTitle, "Borys_Tech-Stack", "The board title is wrong");
+    }
+
+    @Test
+    public void addNewBoardCheck() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
+        ChromeDriver driver = new ChromeDriver();
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+
+        String onBoardTitle = driver.findElementByXPath(".//input[@class= 'board-name-input js-board-name-input']").getAttribute("value");
+
+        driver.get("https://trello.com/");
```

Comment:
You can add login code to separate method.
Please do this in all test classes where login is required at test start.

Reply:
ok